pages-e2e/features/customEntrypoint/assets/custom-entrypoint.ts

```diff
@@ -1,3 +1,4 @@
+//@ts-nocheck
```

Comment:
For context, this is required since in the e2es we call next-on-pages with `npx` and we don't install it as a dependency (we did try that but the Pages e2es framework wasn't handling that correctly, so we had to pivot to just `npx` 😓)

___

This was actually working without the ts-nocheck comment when it was introduced: https://github.com/cloudflare/next-on-pages/pull/828
My best guess as to why it used to work but it doesn't anymore is that the Vercel CLI would ignore ts files that it didn't recognize but latest versions of the CLI (in the e2e tests we use always the latest Vercel CLI release) do try to compile all the ts files in the project's directory, generating the build issue here (since, legitimately, the next-on-pages dependency could not be solved)